inc/saiacl.h

```diff
@@ -1192,6 +1194,7 @@ typedef enum _sai_acl_entry_attr_t
      *
      * @type sai_acl_field_data_t sai_uint16_t
      * @flags CREATE_AND_SET
+     * @isvlan false
```

Comment:
it is false by default, but i made it explicit on uint16 types for future adding attributes for users to not forget to set it to true if attribute is vlan